### PR TITLE
6/refactorContinuousDeliveryWorkflows

### PR DESCRIPTION
**Pull request by victorggonzalez**
This PR contains the changes introduced to deploy a stack version to the staging environment.
It lacks the versioning and verification proposals as they are not viable at the moment

### CODE REVIEW

**Comment by victorggonzalez** [comment]
I reopen this PR with the latest changes after resolving the bugs that appeared after a week of testing

**Review by dialexo — APPROVED** [review]

File: scripts/compare.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import json
```
Comment (juancarlosfarah) [inline scripts/compare.py:1]: @victorggonzalez, have you checked this file with PEP-8 style? I think there are some style issues (e.g., missing spaces before a }).

**Review by juancarlosfarah — APPROVED** [review]
Looks great! Just added a very minor comment for the Python script for comparing files.

File: scripts/compare.py (hunk 1 above)
Comment (victorggonzalez) [reply scripts/compare.py:1]: Done! Used pycodestyle to perform the check and formatting

**Pull request merged**